src/View/Helper/SchemaHelper.php

```diff
@@ -157,6 +161,13 @@ public function controlOptions($name, $value, $schema) : array
                 'v-datepicker' => '',
                 'time' => 'true',
             ];
+        } elseif ($type === 'enum') {
+            return [
+                'type' => 'select',
+                'options' => Hash::map($schema['enum'], '{n}', function ($v) {
```

Comment:
In this case `Hash::map` is not required and a little overkill, one may use `array_map` as well:

```php
array_map(
    function ($value) {
        return ['value' => $value, 'text' => $value];
    },
    $schema['enum']
)
```

Also: maybe `text` could be `Inflector::humanize($value)`?